pkg/performanceprofile/controller/performanceprofile/components/machineconfig/machineconfig.go

```diff
@@ -171,7 +173,12 @@ func getIgnitionConfig(profile *performancev2.PerformanceProfile) (*igntypes.Con
 
 	// add rps udev rule
 	rpsRulesMode := 0644
-	rpsRulesContent, err := assets.Configs.ReadFile(filepath.Join("configs", udevRpsRules))
+	var rpsRulesContent []byte
+	if profileutil.IsRpsEnabled(profile) {
```

Comment:
minor nonblocking: similarly, this should hint we want to enable the RPS for all the devices.